lib/Ninja/CMakeLists.txt

```diff
@@ -4,3 +4,5 @@ add_llbuild_library(llbuildNinja
   ManifestLoader.cpp
   Parser.cpp
   )
+
+target_link_libraries(llbuildNinja llvmSupport curses)
```

Comment:
That's a good point: I've now moved the curses dependency to the `unittests/Ninja/NinjaTests` target rather than `lib/Ninja/llbuildNinja`.